DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: sircovid
 Title: SIR Model for COVID-19
-Version: 0.11.15
+Version: 0.11.16
```

Comment:
depending when this is merged may need to upgrade again because of Ed's PR on IFR